articles/140_topic_and_service_name_mapping.md

```diff
@@ -219,13 +219,83 @@ Since ROS 2 topic and service names are expanded to fully qualified names, any b
 Additionally any URL related syntax, e.g. the `rostopic://` prefix, will be removed once parsed.
 Therefore only forward slashes have to be substituted when converting to DDS topic names.
 
-### Substitution of the Namespace Delimiter
+### DDS Partitions as namespace hierarchy
 
-The namespace delimiter in ROS 2 topic and service names, a forward slash (`/`), will be replaced with double underscores (`__`).
-Note that as fully qualified ROS 2 topic and service names are absolute, there is always a leading forward slash (`/`).
+Having looked closer to the various QoS settings inside the DDS protocol, paritions have been considered as the prefered method of implementing namespaces.
+DDS partitions allow up to 64 items in a string array, limiting the accumulated string size to 255 characters (TODO: Find spec ref).
+The big benefit of handling the namespaces with partitions is that it generally stays ROS agnostic.
+This means if future implementations of the RMW interface (e.g. ZeroMQ etc.) want to introduce another, different namespace system, they are non-restricted.
+
+#### DDS Partitions Characteristics
```

Comment:
I think this is where it would be good to mention character restrictions on DDS partitions (no regex special characters from what I read). I would also suggest moving the length limit to this section.